### ISSUE TITLE
How to change language from russia to english in Driver Pack Solution 17?

### ISSUE THREAD

**Issue by dileepkumarbattula** [DriverPackSolution/DriverPack-Solution#64]
i need to change language from russia to english in Driver Pack Solution 17.

**Comment by ahyanadm** [comment]
i need to change language from russia to english in Driver Pack Solution 17.
how?

**Comment by edwin270** [comment]
I had to change the main JavaScript language files into English personally, that is what I use.

**Comment by dileepkumarbattula** [comment]
Dear edwin270
can you please tell me how to change it to english language?

**Comment by ha90zem** [comment]
to change language \bin\Tools\language open EN.js then open DRP 

good luck :) :+1:

**Comment by edwin270** [comment]
Have attached my English mod, just extract and replace the folder in the root of Driverpack, please do make a backup, will not be held liable for any error (everything works though).

[bin.zip](https://github.com/DriverPackSolution/DriverPack-Solution/files/72487/bin.zip)

**Comment by ivex001** [comment]
Well i didn't understand what ha90zem said about opening "EN.js and then open DRP". The en.js just run and nothing happens. edwin270 thanks but your english mod has only 6 phrases translated. Doesnt help much but thank you anyway.

**Comment by edwin270** [comment]
Seriously translated more than just 6 phrases, will check the other files and re send

**Comment by ivex001** [comment]
Yes you're right. Then i dont know what happened with my copy. Only six strings showing translated.  See for yourself: http://s13.postimg.org/tp46blmt3/Captura_de_Ecr_1.png

**Comment by parthmangi6795** [comment]
Does any one knows how to translate DRP 17.1.0 language from russian to english ??

**Comment by dkw789** [comment]
(partial fix) To change language from russia to english in Driver Pack Solution 17.
1/  In \bin\Tools\language change ru.js to ru.bak or just delete it.
2/ then rename  EN.js  to  change ru.js 
then open DRP

**Comment by ivex001** [comment]
Did you actually try it? I did that but the others languages appears to bo incomplete!

**Comment by dkw789** [comment]
I did say **_# "partial_** fix"  and yes not all of it is translate to english yet :(

**Comment by azharisan** [comment]
i say fuck with trojan 
![screenshot_76](https://user-images.githubusercontent.com/40834416/43699552-d26a578e-9981-11e8-9ede-a0affeb9372d.png)

drp.su ???
